cypress/e2e/systemIntake.cy.js

```diff
@@ -134,10 +129,10 @@ describe('The System Intake Form', () => {
 
     // Submit
     cy.contains('button', 'Send my Intake Request').click();
-    cy.contains('h1', 'Your Intake Request has been submitted');
+    cy.contains('h1', 'Success!');
   });
 
-  it('displays and fills conditional fields', () => {
+  it.only('displays and fills conditional fields', () => {
```

Comment:
Can you add to this e2e test so that it checks for the annual cost and contract fields in the review step?